### PR TITLE
fix: [M3-8740] - Convert Object Storage size from `GiB` to `GB` in frontend

### PR DESCRIPTION
**Pull request by hasyed-akamai**
## Description 📝

- This PR changes the Object Storage bucket size numbers from `GiB` to `GB` in the frontend to avoid confusion during migration.

- Previously, the size numbers were displayed in `GiB`, and now they are displayed in `GB`.

## Changes  🔄

- Converts Object Storage bucket size display from `GiB` to `GB` in the frontend.

## Target release date 🗓️

N/A

## Preview 📷

| Before | After |
| ------ | ----- |
| Before | After |
| Before | After |
| Before | After |
| Before |  @After |

### Prerequisites

- Ensure you have the necessary permissions to create Object Storage Buckets.
- Create a Object Storage Bucket and add a Object to the Bucket.

### Reproduction steps

- Landing:
  - Observe Object Storage bucket size display in Equivalent `GB` in the frontend.

### Verification steps

- After changes Observe Object Storage bucket size display in Equivalent `GB` in the frontend.

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/components/Uploaders/FileUpload.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ export const FileUpload = React.memo((props: FileUploadProps) => {
             })}
             variant="body1"
           >
-            {readableBytes(props.sizeInBytes).formatted}
+            {readableBytes(props.sizeInBytes, { base10: true }).formatted}
```
Comment (cpathipa) [inline packages/manager/src/components/Uploaders/FileUpload.tsx:96]: Nit: You could add a comment to provide context for enabling base10.

**Review by cpathipa — COMMENTED** [review]
@hasyed-akamai You might need to adjust the broken unit tests w.r.t your changes.

File: packages/manager/src/components/Uploaders/FileUpload.tsx (hunk 1 above)
Comment (abailly-akamai) [reply packages/manager/src/components/Uploaders/FileUpload.tsx:96]: Optional: this comment is not super clear. Essentially what you are doing here is converting from binary units (GiB) to decimal units (GB).

**Review by abailly-akamai — APPROVED** [review]

**Review by harsh-akamai — APPROVED** [review]
Changes look good! Thanks @hasyed-akamai

**Pull request merged**